packages/pg-v5/commands/backups/restore.js

```diff
@@ -27,11 +27,11 @@ function * run (context, heroku) {
   let backupURL
   let backupName = args.backup
 
-  if (backupName && backupName.match(/^https?:\/\//)) {
+  if (backupName && /^https?:\/\//.test(backupName)) {
     backupURL = dropboxURL(backupName)
   } else {
     let backupApp
-    if (backupName && backupName.match(/::/)) {
+    if (backupName && /::/.test(backupName)) {
```

Comment:
This could even be `includes()` but I thought I'd leave it for someone else to tackle later